routes/api-v1.php

```diff
@@ -22,3 +23,4 @@
 });
 
 Route::post('generate-signed-url', SignedUrlController::class);
+Route::apiResource('countries', CountryController::class)->only('index');
```

Comment:
@dhrumin-7span Please define get route instead of apiResource

Reply:
Done